src/layer/vector/Path.js

```diff
@@ -71,7 +71,21 @@ export var Path = Layer.extend({
 		// @option bubblingMouseEvents: Boolean = true
 		// When `true`, a mouse event on this path will trigger the same event on the map
 		// (unless [`L.DomEvent.stopPropagation`](#domevent-stoppropagation) is used).
-		bubblingMouseEvents: true
+		bubblingMouseEvents: true,
+
+		// @option keyboard: Boolean = true
+		// Whether the marker can be tabbed to with a keyboard and clicked by pressing enter.
+		keyboard: true,
+
+		// @option title: String= ''
+		// Alternative text for a Figure. Used as `<title>`` in SVG or `aria-label` in Canvas renderer.
+		// Useful for accessibility.
+		title: '',
+
+		// @option description: String= ''
+		// Text describing what is on the figure.
+		// Useful for accessibility.
+		description: '',
```

Comment:
I suppose this option could be named only `desc` to reflect (in the SVG case) the `<desc>` element, similar to how the `alt` option is not named `alternative`~ for the `alt` attribute.

But let's hear what others think.

Reply:
Although this is not as straightforward as `alt` in Marker since it has two implementations depending on a renderer, I think `desc` is descriptive enough and shorter, and I like the proposed change.

Reply:
Yes, let's go ahead and rename it `desc`. 👍🏼